package.json

```diff
@@ -21,6 +21,7 @@
     "c8": "^8.0.1",
     "colorette": "2.0.20",
     "dirname-filename-esm": "1.1.1",
+    "ejs": "^3.1.9",
```

Comment:
why add ejs?

Reply:
You were defaulting to `.ejs` before. 

https://github.com/tinyhttp/tinyhttp/blob/bec8ee50024a645ad26663752b5761dd03508576/packages/app/src/app.ts#L173

I'll remove it